communication.md

```diff
@@ -1,120 +1,128 @@
 
-**I need to preserve non-urgent email for later search.**
+## Selecting the right communication medium
 
-* Send/forward the email to an archive group in [Google Groups](software/googleapps.html). For maximum transparency, we recommend using these archive groups as a BCC recipient in all external email communication.
+Here we describe common situations and the most appropriate medium we use to resolve them. _When in doubt, use common sense._
+
+**I need to pass non-urgent information to one or more persons and don't expect feedback.**
 
-**I need to pass information to one or more persons and get their feedback quickly, e.g. to be able to continue my current work.**
+* If the information is of a short-lived nature, post it to a Slack channel. See also the "Status updates" section.
+* Otherwise post it to a Team Conversation in Asana.
 
-Try these communication channels in order until you can establish a contact:
-[voice/video call](/software/skype.html), [Slack](/software/slack.html) DM.
+**I need to pass non-urgent information to one or more persons and get their feedback.**
 
-**I need to get support immediately, e.g. during incident management.**
+* General: Start a Team Conversation in [Asana](/software/asana.html).
+* In the context of a project: Leave a comment in the relevant [Asana](/software/asana.html) project/task.
 
-* Incident management: Create or escalate the incident in PagerDuty, write an "@channel" message in the \#incident channel on [Slack](/software/slack.html).
+**I need to pass information to one or more persons and get their feedback quickly.**
 
+* @-mention individual persons or address the whole group via `@here` on the appropriate [Slack](/software/slack.html) channel. We expect each other to respond before the end of their work day.
 
-### Town Hall meeting
+**I need to preserve non-urgent email for later search.**
 
-Once a week, we have a [Skype](/software/skype.html) call that everyone in the company is expected to attend, our "Town Hall". This call has the purpose of getting the whole team up-to-date with everything that's relevant:
+* Send/forward the email to an archive group in [Google Groups](software/googleapps.html). For maximum transparency, we recommend using these archive groups as a BCC recipient in all external email communication.
 
-* State of the business
-* Important events
-  * Changes to processes
-  * Important customers we got or lost
-  * Employee birthdays, anniversaries or other news
-* Recent achievements
+**I need to get immediate response on an urgent matter.**
 
-You can find the agenda for upcoming Town Halls on Asana and we encourage you to add your own agenda items that you find relevant to the company. We encourage everyone to also add relevant FYI items, e.g. plans for a longer vacation.
+* @-mention individual persons or address the whole group via `@here` on the appropriate [Slack](/software/slack.html) channel. Preface your post with "URGENT:".
```

Comment:
I think you're too restrictive in your definition of "urgent" and who to contact in these cases.

Similar to my Intercom example above, Simon just now needed quick feedback on a support case because he felt the customer relationship would suffer from a delayed response: http://d.pr/i/maBoWX From my POV, he was perfectly right in addressing me personally and not the channel in general.